### ISSUE TITLE
Extra Utilties healing axe / erosion shovel take damage when stripping wood / making path blocks

### ISSUE THREAD

**Issue by koolkrafter5** [Roadhog360/Et-Futurum-Requiem#564]
### Initial Questions

- [X] I have searched this issue tracker and there is nothing similar already, and this is not on the [list of known issues](https://github.com/Roadhog360/Et-Futurum-Requiem/issues/146). Posting on a closed issue saying the bug still exists will prompt us to investigate and reopen it once we confirm your report.
- [X] I have verified that this issue occurs in a SUPPORTED environment, meaning I can reproduce this WITHOUT OptiFine, and without Bukkit+Forge server software like KCauldron, Thermos, Crucible, Mohist etc

### Mod Version

2.6.2

### Describe the Issue

Extra Utilities's healing axe and erosion shovel are normally supposed to be unbreakable, but they take durability damage when stripping wood / making path blocks:
![image](https://github.com/user-attachments/assets/79b84b83-1157-45fc-b122-51a017789db3)

### Other Details

Another minor problem I found while testing:
Redstone Arsenal's flux-infused axe/shovel do not consume or check for RF to strip logs/make paths.

These seem to work fine:
Unbreakable Tinkers' Construct tools do not take damage from those actions. Blood Magic's bound shovel/axe do not take durability damage as expected. Thaumic Tinkerer's (awakened) ichorium tools also do not take damage (I don't think they even have a durability bar).

Those are all of the unbreakable tools I am aware of, but it could be worth also testing tools from other mods if you know of any.

**Comment by Roadhog360** [comment]
If they're not supposed to take damage, I wonder why they even have durability bars in the first place. Or why they accept the "damage item" code.

I'm not sure whether this issue is on my end because I'm pretty certain that other things would cause the tools to be damaged too.

**Comment by Roadhog360** [comment]
Might forward this to BugTorch

**Comment by Kynake** [comment]
> If they're not supposed to take damage, I wonder why they even have durability bars in the first place. Or why they accept the "damage item" code.
> 
> I'm not sure whether this issue is on my end because I'm pretty certain that other things would cause the tools to be damaged too.

You're correct on that, one example of this is using the healing axe within Ender IO machines.

**Comment by Kynake** [comment]
I'd say this issue can be closed, as this is fixed in Hodgepodge https://github.com/GTNewHorizons/Hodgepodge/pull/602
